packages/devtools_app/lib/src/debugger/codeview.dart

```diff
@@ -398,44 +399,29 @@ class _CodeViewState extends State<CodeView>
     );
   }
 
-  Widget wrapInElevatedCard(
-    Widget widget, {
-    double width = wideSearchTextWidth,
-  }) {
-    return Card(
-      elevation: defaultElevation,
-      color: Theme.of(context).scaffoldBackgroundColor,
-      shape: RoundedRectangleBorder(
-        borderRadius: BorderRadius.circular(defaultBorderRadius),
-      ),
-      child: Container(
-        width: width,
-        height: defaultTextFieldHeight + 2 * denseSpacing,
-        padding: const EdgeInsets.all(denseSpacing),
-        child: widget,
-      ),
-    );
-  }
-
   Widget buildFileSearchField() {
-    return wrapInElevatedCard(
-      FileSearchField(
+    return ElevatedCard(
+      child: FileSearchField(
         debuggerController: widget.controller,
       ),
       width: extraWideSearchTextWidth,
+      height: defaultTextFieldHeight,
```

Comment:
why `defaultTextFieldHeight` now instead of `defaultTextFieldHeight + 2 * denseSpacing` like before?

Reply:
To account for the fact that there is no longer any padding (I think that's what the original calculation was doing, making the height of the container the size of the text height plus the padding). 

With how the file opener is now positioned, I think no padding looks better.

Before:
![Screen Shot 2022-02-01 at 4 53 29 PM](https://user-images.githubusercontent.com/21270878/152076897-cacd8913-c955-4d81-a67f-018361bfbd3f.png)

After:
![Screen Shot 2022-02-01 at 4 53 10 PM](https://user-images.githubusercontent.com/21270878/152076879-3cba4830-6aed-4ee1-9932-13523d667b4d.png)